client/verta/tests/unit_tests/deployment/test_build.py

```diff
@@ -80,6 +82,11 @@ def test_model_version_list_builds(
             ],
             json={"builds": build_dicts},
         )
+        rsps.get(
+            url=model_version_url,
+            status=200,
+            json={"workspace_id": "123"},
+        )
```

Comment:
Added this because the usage of `self.workspace` means that RMV [makes another call out to registry](https://github.com/VertaAI/modeldb/blob/7b48f0570d56f0f6c842a47e812ea25debc796b2/client/verta/verta/registry/entities/_modelversion.py#L176) to get it's workspace, so that call had to be mocked.